balena.yml

```
@@ -0,0 +1,14 @@
+type: type-product-os-t-node-module-source@0.0.3
+version: 1.2.0
+name: Transformer Runtime Package
+data:
+  $transformer:
+    dummy: 1
```

```suggestion
  $transformer: {}
```
better so "dummy" doesn't get passed through *all* contracts